tests/integration/deploy/test_deploy_command.py

```diff
@@ -58,6 +61,15 @@ def tearDown(self):
             # because of the termination protection, do not delete aws-sam-cli-managed-default stack
             if stack_name != SAM_CLI_STACK_NAME:
                 self.cf_client.delete_stack(StackName=stack_name)
+        for stack_name_with_region in self.stack_names_with_regions:
```

Comment:
Any reason, we couldn't combine this into `self.stack_names` instead of creating another list to loop over?

Reply:
To make the minimal changes. `self.stack_names` expects a list of strings (stack names) while  `self.stack_names_with_regions` expects a list of dicts (stack names and regions). We could have refactored `self.stack_names` to expect a list of dicts instead of strings but this will make a broader changes (every line that pushes to the list will change). So, we can do this in a different PR.